Admin/CrossTenantMailboxMigrationValidation.ps1

```diff
@@ -722,23 +722,24 @@ function CheckObjectsSourceOffline {
                     Write-Verbose -Message "Informational: SOURCE mailbox doesn't contain any X500 address"
                 }
 
-                #Check ExternalEmailAddress on TargetMailUser with primarySMTPAddress from SourceMailbox:
+
```

Comment:
Empty line needs to remove for formatting.

Reply:
done